src/components/Questions/QuestionModal.vue

```diff
@@ -305,10 +313,10 @@ export default defineComponent({
         return false
       }
       if (isQuestionTypeSubjective.value) return currentDraftResponse != ""
-      if (typeof currentDraftResponse == "number") {
+      if (isQuestionTypeNumericalInteger.value || isQuestionTypeNumericalFloat.value) {
```

Comment:
Suggestion -- you can also create another computed variable which tells if the question is of numerical type (either numerical-integer or numerical-float). Not necessary though. If you feel it'll make code more readable you can use that

Reply:
then we'll get an error that `isQuestionTypeNumericalInteger` is created but not used :P 
i think its better to have two separate computed vals for integer and float, there could be a future case when we may have to use each separately